caddyhttp/browse/setup.go

```diff
@@ -477,6 +477,16 @@ footer {
 			var filterEl = document.getElementById('filter');
 			filterEl.focus();
 
+			function initFilter() {
+				if (!filterEl.value) {
```

Comment:
This check ensures that any user changes to the initial filter value aren't clobbered when navigating forward/backward.